examples/jest/__tests__/catAPI.spec.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const path = require('path')
+const { Pact, Matchers } = require('../../../dist/pact')
+const getMeCats = require('../').getMeCats
+
+describe("Cat's API", () => {
+  let url = 'http://localhost'
+
+  describe("another works", () => {
+    it('returns a successful body', async () => {
```

Comment:
Ah - the use of `async` is failing the node 6 build. To provide the best experience for newcomers, we'd like to support node 6 without transpiling. Can you update the PR?

Reply:
I reckon it's worth keeping node 6 in CI at least until it's EOL'ed (and possibly for a bit after).